examples/User Creation & Provisioning/README.md

```diff
@@ -35,18 +35,27 @@ This PowerShell script uses the Box CLI to build and create an onboarding folder
 
 ## Configure the script
 ### Update the list of employee accounts for creation
+
+The header row should look like the below:
+
+firstName,lastName,email,username
+
+NOTE 1 - EMAILS MUST BE UNIQUE ACROSS ALL OF BOX - CANNOT CREATE EMAILS USED PREVIOUSLY
+
+NOTE 2 - USERNAME MUST BE UNIQUE ACROSS YOUR BOX INSTANCE. - THIS IS USED FOR THE PERSONAL FOLDER NAME
+
 For example, update the [employees_5.csv](/examples/User%20Creation%20%26%20Provisioning/Employees_5.csv) with the following data:
 ```
-firstName,lastName,email
-Managed,User 1,ManagedUser1@test.com
-Managed,User 2,ManagedUser2@test.com
-Managed,User 3,ManagedUser3@test.com
+Managed,User 1,ManagedUser1@test.com,manageduser1
+Managed,User 2,ManagedUser2@test.com,manageduser2
+Managed,User 3,ManagedUser3@test.com,manageduser3
 ```
 ### List of parameters
 
 - `EmployeeList`: Path to Employee List CSV.
-- `RootFolderParentID`: Destination folder ID for your changes, either when using JSON file as input to create folder structure, or uploading local structure. It is set to `0` by default, but feel free to set it to your needs.
-- `FolderStructureJSONPath`: Your own Folder Structure JSON Path. You can also change the `RootFolderName`. It's a name of the folder, that will be created as parent for folders from JSON structure. It's set to `Onboarding` by default, but feel free to set it to your needs.
+- `PersonalFolderParentID`: Destination folder ID for all personal folders to be created in, either when using JSON file as input to create folder structure, or uploading local structure. This folder should be made prior to running the script the first time. It is not advised to make this value 0, as this will create individual Personal folders in root of the account you set up the cli with.
+- `FolderStructureJSONPath`: Your own Folder Structure JSON Path. 
+- `PersonalFolderSlug`. Ending name of the folder that will be created as parent for folders from JSON structure. It's set to `Personal Folder` by default, but feel free to set it to your needs. The username is concatenated with this value to create each user's personal folder name. ex - `rsmith2's Personal Folder`.
```

Comment:
As Minh mentioned, `$PersonalFolderSlug = ""` isn't defaulted to `Personal Folder`, but defaults to empty string.